src/layout/stack.scss

```diff
@@ -0,0 +1,119 @@
+// Stack layout helper
+
+.Stack {
+  --Stack-gap: var(--primer-stack-gap-normal, 16px);
+  --Stack-gap-whenNarrow: var(--Stack-gap);
+
+  display: flex;
+  flex-flow: column;
```

Comment:
I think it makes more sense if the default `flex-direction` uses the initial `row` value

Reply:
@josepmartins I believe the block/vertical direction will be the most common application for this component. Also, the name "Stack" is usually referred to piles of objects that appear one on top of the other 😛